drafts/Asynchronous-programming-made-resilient-with-MPFT.adoc

```diff
@@ -0,0 +1,230 @@
+---
+layout: post
+title: "Asynchronous programming made resilient with MicroProfile Fault Tolerance"
+categories: blog
+author_picture: https://avatars2.githubusercontent.com/u/55994281?s=460&u=e02b856ba160c3a4275f760606f679811c90a1af&v=4
+author_github: https://github.com/Joseph-Cass
+seo-title: Using MicroProfile Fault Tolerance @Asynchronous Open Liberty - OpenLiberty.io
+seo-description: MicroProfile Fault Tolerance @Asynchronous allows you to add additional functionality to @Timeout and @Bulkhead annotations and allows you to use the other Fault Tolerance annotations with other asynchronous APIs.
+blog_description: "MicroProfile Fault Tolerance @Asynchronous allows you to add additional functionality to @Timeout and @Bulkhead annotations and allows you to use the other Fault Tolerance annotations with other asynchronous APIs"
+additional_authors: 
+ - name: Andrew Rouse
+   github: https://github.com/Azquelt
+   image: https://avatars3.githubusercontent.com/u/2011102?s=460&u=4754ac398d208d72a7aff0bdeab828562a192d53&v=4
+---
+= Asynchronous programming made resilient with MicroProfile Fault Tolerance
+Joseph Cass <https://github.com/Joseph-Cass>
+
+https://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html[Java 8] is long-term support and https://www.jetbrains.com/lp/devecosystem-2019/java/[the most commonly used Java version]. It has many awesome features; one of which is https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletionStage.html[CompletionStage], which allows you to make your code https://www.oracle.com/technetwork/database/application-development/jdbc/learnmore/dev4798-5180524.pdf[asynchronous]. Great stuff! However, you might have the following questions:
```

Comment:
> It has many awesome features; one of which is CompletionStage, which allows you to make your code asynchronous.

I think this is a misleading statement. How about "`CompletionStage`, which provides new ways to write asynchronous code."

Reply:
Yes I agree. It currently implies that this wasn't possible before `CompletionStage`